client/state/Makefile

```diff
@@ -5,6 +5,9 @@ BASE_DIR := $(NODE_BIN)/../..
 NODE_PATH := $(BASE_DIR)/client
 
 test:
-	@NODE_ENV=test NODE_PATH=$(NODE_PATH) $(MOCHA) --compilers jsx:babel/register,js:babel/register --reporter $(REPORTER)
+	@NODE_ENV=test NODE_PATH=$(NODE_PATH) $(MOCHA) --compilers jsx:babel/register,js:babel/register --reporter $(REPORTER) $(addprefix --grep ,$(GREP))
```

Comment:
@gwwar: I think we might be able to do something with `package.json` scripts, as `npm` will automatically traverse up to the closest `package.json`. The idea would be to specify the `test` as running Mocha with all of our environment variables. This could be in a `package.json` in `client/state`, or in the root `package.json`, so long as we detect whether the current working directory is outside the project root.

Then, running `cd client/state/sites/plans && npm test` would only run the site plans tests.